ACNHMS_Source/Assets/Data/VillagerTable.asset

```diff
@@ -0,0 +1,49 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 0}
+  m_Name: VillagerTable
+  m_EditorClassIdentifier: Assembly-CSharp:ACNHMS.Sebbett:VillagerList
+  anteaters: []
+  bears: []
+  birds: []
+  bulls: []
+  cats: []
+  cubs: []
+  chickens: []
+  cows: []
+  alligators: []
+  deer: []
+  dogs: []
+  ducks: []
+  elephants: []
+  frogs: []
+  goats: []
+  gorillas: []
+  hamsters: []
+  hippos: []
+  horses: []
+  koalas: []
+  kangaroos: []
+  lions: []
+  monkeys: []
+  mice: []
+  octopuses: []
+  ostriches: []
+  eagles: []
+  penguins: []
+  pigs: []
+  rabbits: []
+  rhinos: []
+  sheep: []
+  squirrels: []
+  tigers: []
+  wolves: []
```

Comment:
If .asset files will be present in the final PR, ensure your project builds correctly for all 4 platforms for release.

Reply:
Thaaaat file shouldn't even be there anymore, that was from when I was trying different methods. I'll clean that up.